bangazonapi/views/paymenttype.py

```diff
@@ -79,12 +79,9 @@ def destroy(self, request, pk=None):
 
     def list(self, request):
         """Handle GET requests to payment type resource"""
-        payment_types = Payment.objects.all()
-
-        customer_id = self.request.query_params.get('customer', None)
-
-        if customer_id is not None:
-            payment_types = payment_types.filter(customer__id=customer_id)
+        auth_customer = Customer.objects.get(user=request.auth.user)
+        payment_types = Payment.objects.filter(
```

Comment:
Take another look at this one.. 

When I'm logged in as Steve, and I request `/paymenttypes` w token 9ba45f09651c5b0c404f37a2d2572c026c146690, I get a single payment type back in the response w the id of 1, and the account number 24ijio68948fj8439. However, Steve doesn't have any payment types saved in the db. Payment type 1 is actually associated with customer 5, Joe Shepherd. 

Look closely at the relationship between tokens/users, users/customers, and customers/paymettypes to identify the problem and see if you can come up with a solution.